src/main/java/io/quarkus/search/app/indexing/IndexingConfig.java

```diff
@@ -9,11 +9,22 @@
 interface IndexingConfig {
     OnStartup onStartup();
 
+    Scheduled scheduled();
+
     interface OnStartup {
         @WithDefault("true")
         boolean enabled();
 
         @WithDefault("3s")
         Duration waitInterval();
     }
+
+    interface Scheduled {
+        @WithDefault("true")
+        boolean enabled();
+
+        // At 19:00:00 every day
+        @WithDefault("0 0 19 * * ?")
```

Comment:
This probably makes more sense to put in `application.properties`, given how specific it is?
You can have properties without defaults.